### PR TITLE
Brand Page

### PR DESCRIPTION
**Pull request by Hornebom**
Ticket
https://app.zenhub.com/workspaces/centrifuge-applications-5fa29e924059e5001625e2a3/issues/centrifuge/website/502

### CODE REVIEW

File: src/pages/brand.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    }
+  }
+}
+
+export default function Brand({ data }: BrandProps) {
+  const { brands, title } = data.dataJson
+
+  return (
+    <Layout>
+      <h1 className="sr-only">{title}</h1>
```
Comment (onnovisser) [inline src/pages/brand.tsx:44]: can be a styled component instead of global classname

File: src/components/brand-section/Colors.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  return (
+    <Shelf
+      backgroundColor={value}
+      px={2}
+      pt={8}
+      pb={1}
+      alignItems="end"
+      gap={2}
+      style={{ border: outline ? `1px solid ${colors.borderPrimary}` : 0 }}
+      maxWidth={sizes.containerNarrow}
```
Comment (onnovisser) [inline src/components/brand-section/Colors.tsx:57]: can be `maxWidth="containerNarrow"`

File: src/components/brand-section/Colors.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+  return (
+    <Shelf
+      backgroundColor={value}
+      px={2}
+      pt={8}
+      pb={1}
+      alignItems="end"
+      gap={2}
+      style={{ border: outline ? `1px solid ${colors.borderPrimary}` : 0 }}
```
Comment (onnovisser) [inline src/components/brand-section/Colors.tsx:56]: a bit more verbose, but perhaps nicer
```
borderStyle="solid"
borderWidth={outline ? 1 : 0}
borderColor="borderPrimary"
```

Comment (Hornebom) [reply src/components/brand-section/Colors.tsx:56]: refactored using props

File: src/components/brand-section/Colors.tsx (hunk 2 above)
Comment (Hornebom) [reply src/components/brand-section/Colors.tsx:57]: refactored using keyword

File: src/pages/brand.tsx (hunk 1 above)
Comment (Hornebom) [reply src/pages/brand.tsx]: refactored to use `<SROnly />` component

Comment (onnovisser) [reply src/pages/brand.tsx]: can we call it `VisuallyHidden` instead?

Comment (Hornebom) [reply src/pages/brand.tsx]: yes, renamed to `VisuallyHidden`

**Review by onnovisser — APPROVED** [review]

**Pull request merged**